autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleListItem/ScheduleListItem.tsx

```diff
@@ -3,6 +3,8 @@ import { useSelector, useDispatch } from 'react-redux';
 import {
   ListItem, ListItemText, ListItemSecondaryAction,
 } from '@material-ui/core';
+import SchoolIcon from '@material-ui/icons/School';
```

Comment:
```suggestion
import HonorsIcon from '@material-ui/icons/School';
```
so that we know what it does

Reply:
Hey Adel, how should I fix this?

Reply:
Just replace `SchoolIcon` with `HonorsIcon`